corehq/form_processor/migrations/0096_add_partial_index_cases.py

```diff
@@ -0,0 +1,26 @@
+# Generated by Django 3.2.20 on 2023-10-05 18:52
+
+from django.db import migrations, models
+
+INDEX_NAME = 'form_processor_co_be9b2037_idx'
+TABLE_NAME = 'form_processor_commcarecasesql'
+COLUMN = 'deleted_on'
+
+CREATE_INDEX_SQL = "CREATE INDEX CONCURRENTLY IF NOT EXISTS {} ON {} ({}) WHERE {} IS NOT NULL".format(
+    INDEX_NAME, TABLE_NAME, COLUMN, COLUMN)
+DROP_INDEX_SQL = "DROP INDEX CONCURRENTLY IF EXISTS {}".format(INDEX_NAME)
+
+
+class Migration(migrations.Migration):
```

Comment:
I think this class should have an `atomic = False` attribute.

Reply:
oops sorry Daniel, I completely missed this last week!